src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -460,15 +460,18 @@ private void AddUserDefinedFunctions(IEnumerable<UDF> parsedUdfs, ReadOnlySymbol
 
             foreach (var udf in udfs)
             {
-                Config.SymbolTable.AddFunction(udf);
                 var binding = udf.BindBody(nameResolver, new Glue2DocumentBinderGlue(), BindingConfig.Default, Config.Features);
 
                 List<TexlError> bindErrors = new List<TexlError>();

```

Comment:
>bindErrors [](http://example.com/codeflow?start=32&length=10)

are these just binding errors? Or any errors (parse, etc). 

(I think this fixes a bug we had previously)

Reply:
just binding errors